libraries/renderer/render_thread.cpp

```diff
@@ -45,4 +51,5 @@ namespace bnb::render
         m_renderer.reset();
         glfwMakeContextCurrent(nullptr);
     }
-} // bnb::render
+
+} /* namespace bnb::render */
```

Comment:
Please apply the style of Banuba SDK by applying ./scripts/reformat.py --docker
- Copy code to the Banuba SDK folder
- Update reformat script to process copied folder
- Run script
- Copy changed files back.

Reply:
To mark the end of namespace '}' it is used the // comment style (one line)